src/env.ts

```diff
@@ -0,0 +1,10 @@
+export let TheReadablStream: typeof ReadableStream;
+export let TheTextDecoder: typeof TextDecoder;
```

Comment:
@alisabzevari I need some help with this. The type of TextDecoder is different from util.TextDecoder. Do you have any idea how to fix this ?